### PR TITLE
rename app_ecosystem_v2 to app_api

### PR DESCRIPTION
**Pull request by szaimen**
Follow https://github.com/nextcloud/server/pull/40295

### CODE REVIEW

File: php/templates/containers.twig [hunk 1]
```diff
@@ -575,9 +575,9 @@
                             {#<input type="checkbox" id="onlyoffice" name="onlyoffice"><label for="onlyoffice">OnlyOffice</label><br>#}
                         {% endif %}
                         {% if is_docker_socket_proxy_enabled == true %}
-                            <input type="checkbox" id="docker-socket-proxy" name="docker-socket-proxy" checked="checked"><label for="docker-socket-proxy">Docker Socket Proxy (needed for <a href="https://github.com/cloud-py-api/app_ecosystem_v2#nextcloud-application-ecosystem-v2">Nextcloud Application Ecosystem V2</a>)</label><br><br>
+                            <input type="checkbox" id="docker-socket-proxy" name="docker-socket-proxy" checked="checked"><label for="docker-socket-proxy">Docker Socket Proxy (needed for <a href="https://github.com/cloud-py-api/app_ecosystem_v2#nextcloud-application-ecosystem-v2">Nextcloud App Ecosystem V2</a>)</label><br><br>
```
Comment (szaimen) [inline php/templates/containers.twig:578]: Was this also rebranded to Nextcloud App Api or something?

Comment (bigcat88) [reply php/templates/containers.twig:578]: Yeah, AppEcosystemV2 will be called AppAPI from now, we are still in process of renaming...

File: php/templates/containers.twig [hunk 2]
```diff
@@ -575,9 +575,9 @@
                             {#<input type="checkbox" id="onlyoffice" name="onlyoffice"><label for="onlyoffice">OnlyOffice</label><br>#}
                         {% endif %}
                         {% if is_docker_socket_proxy_enabled == true %}
-                            <input type="checkbox" id="docker-socket-proxy" name="docker-socket-proxy" checked="checked"><label for="docker-socket-proxy">Docker Socket Proxy (needed for <a href="https://github.com/cloud-py-api/app_ecosystem_v2#nextcloud-application-ecosystem-v2">Nextcloud Application Ecosystem V2</a>)</label><br><br>
+                            <input type="checkbox" id="docker-socket-proxy" name="docker-socket-proxy" checked="checked"><label for="docker-socket-proxy">Docker Socket Proxy (needed for <a href="https://github.com/cloud-py-api/app_ecosystem_v2#nextcloud-application-ecosystem-v2">Nextcloud App API</a>)</label><br><br>
```
Comment (andrey18106) [inline php/templates/containers.twig:578]: `app_ecosystem_v2` repository name will be renamed to `app_api` too

Comment (szaimen) [reply php/templates/containers.twig:578]: ah okay

**Review by andrey18106 — APPROVED** [review]

**Comment by szaimen** [comment]
This is now released with v7.2.0 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel

**Pull request merged**